routes/web.php

```diff
@@ -26,10 +26,10 @@
             ->name("verification.notice");
 
         Route::get("email/verify/{id}/{hash}", [
-            EmailVerificationController::class,
+            config('filament-breezy.email_verification_controller_path'),
```

Comment:
isn't this a breaking change to use this without default value?
as other (already running) apps that don't have this config will receive null?
or am i wrong?

Reply:
TBH, I didn't test it, just asking, how laravel handling this? does it merge package config with user config?

Reply:
It is. I am reviewing now. Thank you!

Reply:
anytime ^^, thanks for your hard work.